source/devices/AM335X/linux/Release_Specific_Migration_Guide.rst

```diff
@@ -437,9 +432,7 @@ Processor SDK 03.03
 
 -  HDMI-connected monitor no longer takes precedence over built-in LCD
    as a default graphics display for e.g. Matrix
--  See the `Release
-   Notes <http://processors.wiki.ti.com/index.php/Processor_SDK_Linux_Release_Notes>`__
-   for full list of modifications.
+-  See the Release Notes for full list of modifications.
```

Comment:
There's a ref for release notes you can use here

Reply:
But that release note corresponds to the release notes of older versions that are not available, so I removed the links.

Reply:
If that content is truly unreachable then all references to it should be purged as well. Drop the note about " -  See the Release Notes for full list of modifications." as there's nothing to see.